EXAMPLES/src/caching/Pagination.re

```diff
@@ -0,0 +1,20 @@
+let cache =
+  ApolloClient.Cache.InMemoryCache.(
+    make(
+      ~typePolicies=[|
```

Comment:
Why not work with objects and have a more 1:1 binding?

Reply:
I just didn't see any way to express the types with objects at all besides doing `Js.t('anything)`. This uses Js.Dict.t under the hood. I opted to just forego requiring the user to `Js.Dict.fromArray` in favor of passing the array directly. Do you think it would be better to force that for clarity?

Reply:
One thing that stinks as well with this implementation is that you can't just choose to bypass that and use `Js.t` yourself because it is expecting specific variants to represent the unions in typescript. 🤷

Reply:
I think this is pretty nice actually!